app/models/miq_alert.rb

```diff
@@ -193,16 +193,21 @@ def evaluate(target, inputs = {})
     # If we are alerting, invoke the alert actions, then add a status so we can limit how often to alert
     # Otherwise, destroy this alert's statuses for our target
     invoke_actions(target, inputs) if result
-    add_status_post_evaluate(target, result, inputs[:description])
-
+    add_status_post_evaluate(target, result, inputs[:ems_event])
     result
   end
 
-  def add_status_post_evaluate(target, result, status_description)
+  def add_status_post_evaluate(target, result, event)
+    status_description, severity, url, status_result =
+      if event.respond_to?(:parse_event_metadata)
+        event.parse_event_metadata
+      end
     status = miq_alert_statuses.find_or_initialize_by(:resource => target)
-    status.result = result
+    status.result = status_result.nil? ? result : status_result
```

Comment:
I wanted to override the value, if it is true or false, not if it is nil.
I came to realize that it's not a good idea to override the evaluation result,
and I'll take care of that as part of the evaluation. (removing out of this pr in favor of)